internal/backends/qt/qt_widgets/tableview.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+// #[repr(C)]
+// #[derive(FieldOffsets, Default, SlintElement)]
+// #[pin]
+// pub struct NativeTableViewColumn {
+//     pub x: Property<LogicalLength>,
+//     pub y: Property<LogicalLength>,
+//     pub width: Property<LogicalLength>,
+//     pub height: Property<LogicalLength>,
+//     pub item: Property<i_slint_core::model::TableColumn>,
+//     pub index: Property<i32>,
+//     pub is_selected: Property<bool>,
+//     pub cached_rendering_data: CachedRenderingData,
+//     pub has_hover: Property<bool>,
+//
```

Comment:
leftover?

Reply:
Yes removed that before. I think I missed to add it to the commit. It's removed now.